moto/settings.py

```diff
@@ -69,6 +69,13 @@ def allow_unknown_region() -> bool:
     return os.environ.get("MOTO_ALLOW_NONEXISTENT_REGION", "false").lower() == "true"
 
 
+def lambda_stub_ecr() -> bool:
+    # Whether to stub or mock ecr backend when deploying image based lambdas.
+    # True => don't requiring image presence in moto ecr backend for `create_function`.
+    # False => require image presence in moto ecr backend for `create_function`
+    return os.environ.get("LAMBDA_STUB_ECR", "TRUE").lower() != "false"
```

Comment:
The two options have a different functionality, actually. The bottom variable (`SOME_CONFIG = ..`) is resolved in the beginning, when Moto is loaded/imported. That means that the env var has to be set before everything else - if a user sets/changes the env var afterwards, Moto has no way of knowing that it was updated.

Because the first option is inside a method, the value is resolved everytime the method is called - i.e., users can change the environment variable whenever they please, and Moto will check the value of the env var every time the code is executed.

Which option is better depends on two things: how often does Moto need to know the value (i.e., how often do we need to make the call to `os.environ`), and how likely is it that the user wants to change this value halfway during the test execution.

In our scenario, we can stick to using the function. Moto only needs to check the env var once, during a specific function call, so the overhead is neglible. It is also likely that the user will want to change this value for different tests, to verify different behaviours.